pkg/pxc/app/statefulset/node.go

```diff
@@ -260,6 +267,11 @@ func (c *Node) Volumes(podSpec *api.PodSpec, cr *api.PerconaXtraDBCluster) (*api
 			vol.Volumes,
 			app.GetConfigVolumes("auto-config", "auto-"+ls["app.kubernetes.io/instance"]+"-"+ls["app.kubernetes.io/component"]))
 	}
+	if cr.CompareVersionWith("1.4.0") >= 0 {
+		vol.Volumes = append(
+			vol.Volumes,
+			app.GetSecretVolumes("vault-keyring-secret", "vault-keyring-secret", true))
```

Comment:
it should be configurable from CR (like `sslSecretName`) and default should be `cr.Name. + "-vault"`

Reply:
Done https://github.com/percona/percona-xtradb-cluster-operator/pull/349/commits/75a1572891a667335f92934b97ca5a0dfcfd612a